easybuild/easyconfigs/g/GDAL/GDAL-2.2.0-intel-2017a-Python-2.7.13-HDF5-1.10.1-HDF-4.2.13.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.0'
+hdf5_ver = '1.10.1'
+hdf_ver = '4.2.13'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s-HDF-%s' % (hdf5_ver, hdf_ver)
```

Comment:
This is becoming a bit of a mess...

I would include `Python` & `HDF5` (to keep consistency with existing easyconfigs), but I see no reason to also include `HDF`, especially since it's an uncommon dep?

Reply:
This is HDF4, we have a usergroup that uses HDF4 input files and am under the impression this is not that exotic, see below.  HDF4 is the purpose of this config.   

I wanted to make clear that this version supports HDF5 and HDF4.  

The GDAL website http://trac.osgeo.org/gdal/wiki/HDF devotes a fair amount of space how to build with support for both.  I assume this is of wider interest and thought I make it available.

You have a suggestion on how to clean up the name?

Reply:
Well, first of all, it's quite long, and the `-HDF5-<version>` part is only there because the `netCDF` used here happens to have that versionsuffix...

Including the `-HDF-<version>` in it makes sense to indicate this is a special variant of `GDAL`, but maybe it's overkill... Does also including `HDF` cripple the `GDAL` installation somehow(compared to a GDAL that doesn't, or does it just result in enhanced functionality?

If it's the latter, than I think including the `HDF` version in the versionsuffix is overkill (we don't list the version of any other dependencies either...).